server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/scala/org/apache/james/jmap/rfc8621/contract/EchoMethodContract.scala

```diff
@@ -130,6 +131,7 @@ trait EchoMethodContract {
   }
 
   @Test
+  @Tag("BasicFeature")
```

Comment:
We should have a 

```
inteface Tags {
   String BASIC_FEATURE = "BasicFeature";
}
```

If we always specify the tag as a string there will be copy & paste mistakes ;-)

Reply:
agree